test/unit/au/org/emii/portal/proxying/RequestProxyingControllerTests.groovy

```diff
@@ -9,8 +9,12 @@ package au.org.emii.portal.proxying
 
 import grails.test.*
 
+import static au.org.emii.portal.HttpUtils.Status.HTTP_500_INTERNAL_SERVER_ERROR
```

Comment:
You may include `au.org.emii.portal.HttpUtils.Status.*` here. Wouldn't make much of a difference and less likely to break.